src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,6 +16,27 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
+    @Override
+    public int hashCode() {
+        return getByteSum(this.color) * this.getWidth() * this.getLength();
```

Comment:
try implement this method using `prime number`
and let's use hashcode of color, width and length